lib/datetime_helper.rb

```diff
@@ -2,7 +2,7 @@
 
 module DatetimeHelper
 
-  ZULU_TIME_PATTERN = /^(\d{4})-([0-1][0-9])-([0-3]\d{1})T([0-2]\d{1}):([0-5]\d{1}):([0-5]\d{1})(\.[0-9]{1,3})?Z$/
+  ZULU_TIME_PATTERN = /^(\d{4})-([0-1][0-9])-([0-3]\d{1})T([0-2]\d{1}):([0-5]\d{1}):([0-5]\d{1})(((\.[0-9]{1,3})?Z$)|\+00:00)$/
```

Comment:
this lets `2016-11-09T23:58:09.000Z` pass but not `2016-11-09T23:58:09.000+00:00`. maybe we need to change the parenthesis a bit so milliseconds are allowed for both configurations. repetition of `$` might've be avoided too